packages/unmock-core/src/service/parser.ts

```diff
@@ -0,0 +1,34 @@
+import jsYaml from "js-yaml";
+import { IServiceDef, IServiceDefFile, IServiceParser } from "../interfaces";
+import { Service } from "./service";
+
+const PATTERN_FOR_KNOWN_FILENAMES = /^(?:index|spec|openapi)\.ya?ml$/;
+
+export class ServiceParser implements IServiceParser {
+  public parse(serviceDef: IServiceDef): Service {
+    const name = serviceDef.directoryName;
```

Comment:
This should probably be read from the yaml instead, or fallback to the directory name if not found? But then the spec is not valid openapi unless the name is prefixed with `x-`?

Reply:
Maybe just a `TODO` note here, we'll probably want to collectively decide on the heuristic here.
(Could be read from the yaml file, parsed from the `info.title` OAS, retrieved from the directory name, etc...)